openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenAPICmdTest.java

```diff
@@ -500,6 +500,151 @@ public void testSuccessfulServiceGenerationForYML() throws IOException {
         }
     }
 
+    @Test(description = "Test openapi service generation for single file option")
+    public void testSingleFileServiceGeneration() throws IOException {
+        Path jiraYaml = resourceDir.resolve(Paths.get("jira_openapi.yaml"));
+        String[] args = {"--input", jiraYaml.toString(), "-o", this.tmpDir.toString(), "--mode",
+                "service", "--single-file"};
+        OpenApiCmd cmd = new OpenApiCmd(printStream, tmpDir, false);
+        new CommandLine(cmd).parseArgs(args);
+        try {
+            cmd.execute();
+        } catch (BLauncherException e) {
```

Comment:
Do we throw this `BLauncherException` from `cmd.execute()`?

Reply:
looks like a redundant catch. its in several old test cases as well. will update